app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -72,6 +73,33 @@ private static void upgrade(int oldVersion) {
             }
 
             UserPreferences.setQueueLocked(false);
+
+            // Migrate hardware button preferences to new system that supports repeat actions
```

Comment:
I am unsure if this was done correctly, or if I need to create a new upgrade block for a different version

Reply:
This is correct. It might happen that it isn't executed on your development device but it will when actual users upgrade